conn.go

```diff
@@ -333,6 +341,15 @@ func (c *Conn) SessionID() int64 {
 	return atomic.LoadInt64(&c.sessionID)
 }
 
+// Password returns the current session password of the connection.
```

Comment:
This seems a bit overkill.  Why not just at a RWMutex to Conn?

Reply:
I agree it's simpler, this was to match the way `sessionID` is accessed. I've changed to use a RWLock.